calico_containers/tests/st/docker_host.py

```diff
@@ -19,6 +19,10 @@ def __init__(self, name):
 
         self.ip = docker.inspect("--format", "{{ .NetworkSettings.IPAddress }}",
                                  self.name).stdout.rstrip()
+        ip6 = docker.inspect("--format", "{{ .NetworkSettings.GlobalIPv6Address }}",
+                             self.name).stdout.rstrip()
+        self.ip6 = ip6 or "fd80:24e2:f998:72d6::1"
```

Comment:
Leave a `# TODO` comment that hardcoding is only temporary until we sort out multi-host for IPv6.